src/models/UserModel.js

```diff
@@ -11,6 +11,68 @@ const UserSchema = new Schema({
     type: String,
     required: true,
     unique: false
+  },
+  role: {
+    type: String,
+    required: true,
+    unique: false
```

Comment:
if you want to add a status for user schema, maybe we keep it as a separate field? 

```status: {
    type: Number,
    enum: [0, 1, 2], // or use an object for a more readable code (e.g., { ACTIVE: 0, INACTIVE: 1 })
    default: 0
  },```

Reply:
Yeah, sorry my bad. In the role document,  I would like to add the field `enum: ['admin', 'user']`, along with `default: 'user'`

Additionally, we need a new document for the 'status', which should be defined as follows:

```
status: {
    type: String,
    required: true,
    enum: ['active', 'inactive'],
    default: 'active'
}
```
This approach is more visual and understandable than using numbers.

Reply:
yes, we can do that. Do you want to go back to the data description in the documentation to see if we need to change any data in the other collections, because I'll mostly base on that to create the schema. We can always come back to fix and add more fields once we start the app logic and we find the the data type might need changes. Once again, my focus is to make the models , their relationships and seeding data work. Then once we finish the routes, we have more ideas of what to do with the database.

Reply:
We often discover certain aspects only when we're actively coding, don't we? 

After revisiting the data descriptions in our documentation, I've realised that some field names could be more appropriately named. For instance, in the products collection, `categoryId` should be renamed to `category`, and in the Users collection, `userId` should be changed to `user`. This is because we are referencing these fields to their respective _id and retrieving all related data from them.